.github/workflows/main.yml

```diff
@@ -25,17 +25,17 @@ jobs:
 
       - name: Install Zeek
         run: |
-          (cd /tmp && curl -L -O https://download.zeek.org/binary-packages/xUbuntu_20.04/amd64/zeek${ZEEK_TAG}-core_${ZEEK_VERSION}_amd64.deb)
+          (cd /tmp && curl -L -O https://download.zeek.org/binary-packages/xUbuntu_22.04/amd64/zeek${ZEEK_TAG}-core_${ZEEK_VERSION}_amd64.deb)
           sudo apt install -y /tmp/zeek${ZEEK_TAG}-core_${ZEEK_VERSION}_amd64.deb
           echo "/opt/zeek/bin:$PATH" >> $GITHUB_PATH
 
       - name: Checkout repository
-        uses: actions/checkout@v2
+        uses: actions/checkout@v4
         with:
           submodules: recursive
 
       - name: Set up ccache
-        uses: hendrikmuhs/ccache-action@v1.2.2
+        uses: hendrikmuhs/ccache-action@v1.2.10
```

Comment:
In theory one should be able to use `@v1.2` or even `@v1` so that the next patch or minor version is picked up automatically.

Unfortunately, seems the `v1` and `v1.2` tags of the ccache-action are still pointing at `v1.2.9` and have not been moved to `v1.2.10`.

![Screenshot from 2023-12-01 12-22-03](https://github.com/zeek/zeek-agent-v2/assets/2100509/57c2d663-4721-4fa2-bab4-3fd95ff4fbd9)

cc @hendrikmuhs - was that an oversight?

Reply:
yes, I forgot to slice the tag.

However, I actually think it is best practice to use a fixed tag/commit hash and use e.g. dependabot to keep it uptodate.